code/modules/mob/living/silicon/robot/laws.dm

```diff
@@ -28,6 +28,7 @@
 
 	to_chat(who, SPAN_WARNING("<b>Obey these laws:</b>"))
 	laws.show_laws(who)
+	to_chat(who, SPAN_WARNING("<b>(No law overrides any other law unless explicitly stated; laws refer to the Stationbound unit and not the player)</b>"))
```

Comment:
```suggestion
	to_chat(who, SPAN_WARNING("<b>(No law overrides any other law unless explicitly stated; laws refer to the stationbound unit and not the player)</b>"))
```

Reply:
Whenever I can, I'll commit this. Just gimmie awhile

Reply:
Alright this should be resolved.